internal/types/types.go

```diff
@@ -866,6 +866,16 @@ type User struct {
 	CodyProEnabledAt      *time.Time
 }
 
+// Returns a name for the user. If the user has a display name,
```

Comment:
```suggestion
// Name returns a name for the user. If the user has a display name,
```
![gif](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExdnJ2NGdwbXB2dTh0eG12MWt0d3VrYjN5aGU0c3NtOHV0NXBjcGtnaCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/1xdHOdXcFAeMpURJeM/giphy.gif)